Cargo.toml

```diff
@@ -21,10 +22,6 @@ tracing-futures = "0.2.5"
 tracing-serde = "0.1.3"
 thiserror = "1.0.30"
 
-[dependencies.chrono]
-features = ["serde"]
-version = "0.4.19"
```

Comment:
i don't understand the need for serde from chrono since we were relying on string values for time right from the beginning.